misk-aws/src/main/kotlin/misk/jobqueue/sqs/SqsJob.kt

```diff
@@ -65,10 +64,11 @@ internal class SqsJob(
    *  that duration. With every subsequent retry the duration becomes longer until it hits the max
    *  value of 10hrs.
    */
-   fun delayForFailure(maxRetryCount: Int = 10) {
+  fun delayForFailure() {
```

Comment:
This breaks backwards compatibility - any services actually passing a value here will break. You'll need to create a new method that doesn't take an argument

Reply:
hmm, how about keeping it this way but just not using the value?

Reply:
I definitely wouldn't silently stop using the parameter, that would be very surprising to callers. I think what you'll need to do is have:

```kotlin
@Deprecated("Prefer delayForFailure(), which uses the queue's maxRetryCount")
fun delayForFailure(maxRetryCount: Int = 10) { /** old code */ }

fun delayForFailure() { /** new code */ }
```

This will change the behavior for anyone not explicitly setting a value but I don't think this function is actually used anywhere so that should be fine

Reply:
Apologies, I forgot that `SqsJob` is internal. Since this isn't used anywhere else I think this change is fine as-is